clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go

```diff
@@ -463,13 +468,16 @@ type objectChecker struct {
 	stopCh    chan struct{}
 	status    objectStatus
 	err       error
+	// namespacedName of the object being checked.
+	namespacedName string
```

Comment:
I would call it a key.
While this is namespacedName in all current usecase, this is an implementation detail (e.g. in the future we may want to identify objects by UID, or we will have CRDs that aren't namespaced, or sth).

Reply:
Good point, done.